src/xdocs/property_types.xml

```diff
@@ -325,16 +325,55 @@
           <td>Definition</td>
         </tr>
 
+        <tr>
+          <td><code>alone</code></td>
+          <td>
+            The brace must be alone on the line. For example:
+            <pre>
+    try {
+        ...
+    <b>}</b>
+    finally {
+        ...
+    <b>}</b>
+            </pre>
+          </td>
+        </tr>
+
+        <tr>
+          <td><code>alone_or_singleline</code></td>
+          <td>
+                  The brace must be alone on the line, yet
+                  single-line format of block is allowed.
+                  For example:
+            <pre>
+    // Brace is alone on the line
+    try {
+        ...
+    <b>}</b>
+    finally {
+        ...
+    <b>}</b>
+
+    // Single-line format of block
+    public long getId() { return id; <b>}</b>
+            </pre>
+          </td>
+        </tr>
+
         <tr>
           <td><code>same</code></td>
           <td>
-            The brace should be on the same line as the next part of a multi-block statement
+            Work as alone_or_singleline but the brace should be on the same line
+            as the next part of a multi-block statement
             (one that directly contains multiple blocks: if/else-if/else or try/catch/finally).
```

Comment:
I think we should expand description to say what will happen if there is no multi-block statement as example shows on line 412.
This should also include line 419 to say that we consider end semicolons, parenthesis, etc... as part of the multi-block statement.

Reply:
changed both lines. We should not treat such cases as multi-block. No following multi-block - curly should be alone.